ap_api/callbacks/customer/index.mdx

```diff
@@ -5,16 +5,19 @@ order: 0
 
 import {EndpointsTable} from "@site/src/components/EndpointsTable";
 
+Customers callback are designed to work with a customer data. When working with the Anchor Platform, business will store KYC data defined by [SEP-12].
```

Comment:
English tweaks, write as: "Customer callbacks are designed to work with customer data. When working with the Anchor Platform, businesses will store KYC data defined by [SEP-12]"